napkon_string_matching/terminology/loinc_search.py

```diff
@@ -12,9 +14,56 @@
 RESPONSE_NO_ENTRIES = "Keine passenden Einträge gefunden"
 RESPONSE_LOGIN = "Log In ‹ LOINC — WordPress"
 
+STOP_WORDS = set(stopwords.words('german'))
+
+EXAMPLEOUTCOME_ITEMS = [
+    """Dialyse:""",
+    """Invasiver Beatmung:""",
+    """Sauerstoff:""",
+    """Trachealkanüle:""",
+    """Respiratorisches Outcome (Sauerstoff-Therapie):"""]
+
+EXAMPLE_QUESTIONS = [
+    """Wurde der Barthel-Index vor der Entlassung durchgeführt?""",
+    """Fatigue Screening durchgeführt? Datum der Durchführung""",
+    """Wurde der "PROMIS Dyspnoe-Funktionseinschränkungen" durchgeführt?"""]
+
 logger = logging.getLogger(__name__)
 
 
+def make_alphanumeric(token: str):
```

Comment:
Maybe choose a name that expresses that this function removes non-alphanumeric values. Currently for it sounds like this function converts them.